cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -332,6 +332,18 @@ data:
       </metric>
     </match>
 
+    # TODO(instrumetation): Reconsider this workaround later.
+    # Trim the entries which exceed slightly less than 100KB, to avoid
+    # dropping them. It is a necessity, because Stackdriver only supports
+    # entries that are up to 100KB in size.
+    <filter kubernetes.**>
+      @type record_transformer
+      enable_ruby true
+      <record>
+        log ${record['log'].length > 100000 ? "#{record['log'][0..100000]}[trimmed]" : record['log']}
```

Comment:
A suffix will cause the whole line to be parsed before we realize it's malformed JSON and revert to text. A prefix would short-circuit that, so it's more efficient. You could add both if it makes for better human readability (e.g., "`[Trimmed long entry] {"a": "very long string...`")...

Reply:
Ack, agree, thanks